### PR TITLE
Add README.md to `model`

### PR DESCRIPTION
**Pull request by dimas-b**
... with API v2 highlights

### CODE REVIEW

File: .idea/codeStyles/codeStyleConfig.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <component name="ProjectCodeStyleConfiguration">
   <state>
-    <option name="PREFERRED_PROJECT_CODE_STYLE" value="GoogleStyle" />
+    <option name="PREFERRED_PROJECT_CODE_STYLE" value="dremio" />
```
Comment (ajantha-bhat) [inline .idea/codeStyles/codeStyleConfig.xml:3]: nit: Can it be a separate PR? As the changes are independent. 

Also, I remember spotless uses GoogleStyle. So, it makes sense for Idea also to use GoogleStyle?

Comment (dimas-b) [reply .idea/codeStyles/codeStyleConfig.xml]: oops - my mistake - removed this change from PR. Thanks for spotting! :+1:

File: model/README.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+However, `org.projectnessie.api` classes are not expected to be used for compiling client-side code outside the scope
+of this repository.
+
+Clients built in `java` should build against the API of the `clients/client` module.
+
+Clients build in other languages can generate language-specific bindings from the published OpenAPI specification of
+the Nessie API.
+
+The published version of the Nessie OpenAPI specification is available on [SwaggerHub](https://app.swaggerhub.com/apis/projectnessie/nessie).
```
Comment (ajantha-bhat) [inline model/README.md:29]: This still points to v1 and in the next release it will be updated to V2 right?
https://app.swaggerhub.com/apis/projectnessie/nessie/0.45.0

**Review by ajantha-bhat — APPROVED** [review]
Overall LGTM. 

Just a question about Swagger hub API version.

File: model/README.md (hunk 2 above)
Comment (dimas-b) [reply model/README.md:29]: v2 OpenAPI has not been released yet. The next release will have both v1 and v2 in Nessie's OpenAPI yaml.

Comment (ajantha-bhat) [reply model/README.md:29]: I thought so. Thanks for the clarification.

**Review by snazy — APPROVED** [review]
